components/NavBar.tsx

```diff
@@ -1,28 +1,45 @@
+"use client";
 import Image from "next/image";
 import Link from "next/link";
-import { Lobster } from "next/font/google";
+import { Lobster, Inter } from "next/font/google";
+import { useSession } from "next-auth/react";
 
 const lobster = Lobster({ weight: "400", subsets: ["latin"] });
-import Icon from "./Icon";
+const inter = Inter({ weight: "400", subsets: ["latin"] });
```

Comment:
this is in your root layout. you shouldn't have to add this font anywhere else since you put it on the body